src/routes/Issues/components/Issues.tsx

```diff
@@ -69,7 +69,11 @@ const getParamsFromProps = (props: IIssuesProps): IParams => {
         ? [project_id]
         : project_id
       : undefined,
-    ordering
+    ordering: ordering 
+      ?typeof ordering === "string"
```

Comment:
Check if ordering is an array if it is, return the first element else return ordering

Reply:
Done :)